config.sample.yaml

```diff
@@ -258,6 +258,10 @@ ircService:
         # insert the IRC domain.
         # Optional. Default: "$NICK (IRC)". Example: "Alice (IRC)"
         displayName: "$NICK (IRC)"
+        # Number of tries a client can attempt to join a room before the request
+        # is discarded. Set to 0 to never retry.
```

Comment:
Can we make it 0 to be infinite, -1 to disable? Default to 5 (which is what is implied by the sample, but not listed as the default).

Reply:
Sure. Makes sense to me :)